src/templates/atoms.njk

```diff
@@ -1348,18 +1371,23 @@
                         <input type="text" name="text-field-error" id="text-field-error" placeholder="Placeholder" />
                     </div>
                     <div class="a-input__info">
-                        <small class="a-input__error-text">Error description</small>
+                        <small class="a-input__error-text">
+                            {{ icon.render('alert-triangle') }} Error description
+                        </small>
                     </div>
                 </div>
             </div>
             <div>
-                <pre class="a-pre a-pre--scrollable"><code class="html">&lt;div class=&quot;a-input has-error&quot;&gt;
-  &lt;label class=&quot;a-input__label&quot; for=&quot;text-field-error&quot;&gt;Error text field&lt;/label&gt;
-  &lt;div class=&quot;a-input__wrapper&quot;&gt;
-    &lt;input type=&quot;text&quot; name=&quot;text-field-error&quot; id=&quot;text-field-error&quot; placeholder=&quot;Placeholder&quot; /&gt;
+                <pre class="a-pre a-pre--scrollable"><code class="html">&lt;div class="a-input has-error"&gt;
+  &lt;label class="a-input__label" for="text-field-error"&gt;Error text field&lt;/label&gt;
+  &lt;div class="a-input__wrapper"&gt;
+    &lt;input type="text" name="text-field-error" id="text-field-error" placeholder="Placeholder" /&gt;
   &lt;/div&gt;
-  &lt;div class=&quot;a-input__info&quot;&gt;
-    &lt;small class=&quot;a-input__error-text&quot;&gt;Error description&lt;/small&gt;
+  &lt;div class="a-input__info"&gt;
+    &lt;small class="a-input__error-text"&gt;
+      &lt;span class="ai" aria-hidden="true"&gt;&lt;svg&gt;&lt;use href="https://cdn.antwerpen.be/core_branding_scss/5.0.1/images/ai.svg#ai-triangle" /&gt;&lt;/svg&gt;&lt;/span&gt;
```

Comment:
Hier zouden we eens moeten kijken hoe we dat best aanpakken, want in principe werkt dit niet (SVG geeft CORS problemen).